ref-exchange/src/account_deposit.rs

```diff
@@ -238,6 +313,14 @@ impl Contract {
         self.accounts.insert(&account_id, &account.into());
     }
 
+    /// save token to owner account as lostfound, no need to care about storage
+    pub(crate) fn internal_lostfound(&mut self, token_id: &AccountId, amount: u128) {
+        let mut lostfound = self.internal_unwrap_or_default_account(&self.owner_id);
+        lostfound.deposit(token_id, amount);
```

Comment:
In theory this can be abused as well. Since `lostfound` doesn't pay for storage an attacker may create a spam of random tokens and deposit them to lostfound. Once a token is deposited the attacker can release their storage and continue with the next token.

How about only whitelisted tokens are going to be deposited to `lostfound` account?

Reply:
The lost-found is mainly for instant-swap feature to ensure swap-out token won't loss if user encounter transfer errors and not registered to our contract. In this case, we keep his swap-out token in lost-found, he can claim a token refund by submitting an off-line application like a DAO proposal. So, we have to ensure all tokens can be stored in lost-found.  

To address the attack you mentioned, maybe we can forbid any public deposit (ft_transfer_call) to lost-found, only leave it to inner deposit from callback of token_withdraw.

Reply:
An attacker may establish lots of pools with spam tokens and gen lots of swap-out transfer error using instant swap for those pool. But create pool will costs storage deposit and I can add additional protect like this to let pool creator pay for lost-found storage: 
```rust
fn internal_add_pool(&mut self, pool: Pool) -> u64 {
        let prev_storage = env::storage_usage();
        let id = self.pools.len() as u64;
        self.pools.push(&pool);
        // to prepare for lostfound storage
        for token in pool.tokens() {
            self.internal_lostfound(token, 0);
        }
        self.internal_check_storage(prev_storage);
        id
    }
```

Reply:
Now we choose to only let whitelisted tokens that can be deposited to lostfound account. From logic side, It's more clear and neat.